src/components/synopsis-report-summer-summary/synopsis-report-summer-summary.js

```diff
@@ -56,7 +56,7 @@ class SynopsisReportSummerSummary extends React.Component {
 
     <p>Hello Team ${studentName} - please see ${studentName}&#39;s Rainier Athletes update over the past week to ensure everyone is aligned and up to date around ${studentName}&#39;s progress on and off the field.</p><br>
 
-    <p>${studentName} ${sr.Summer_weekly_connection_made__c === 'Yes' ? 'met' : 'did not meet'} for check-in this week.</p><br>`;
+    <p>${studentName} ${sr.Summer_weekly_connection_made__c === 'Yes' ? 'yes' : 'no'} for check-in this week.</p><br>`;
```

Comment:
So, two things.
For general understanding of what's going on here, the ? expression is selecting text to include in a sentence based on the value of, in this case, Summer_weekly_connection_made__c. The 'met' and 'did not meet', in other words, are used in a sentence like "Arty did not meet for check-in this week."  So your change would make the sentence read "Arty no for check-in this week."  Not what we want. 

Furthermore, this is not the property/variable we're concerned with.  Refer to the email thread.  We're looking to fix {sr.Identity_Journal_Status__c === 'Met' ? ' turned in' : ' did not turn in'} in synopsis-report-summary.js

Please undo this change.